### ISSUE TITLE
注释和代码不一致

### ISSUE THREAD

**Issue by jiaqianjing** [PaddlePaddle/Paddle#46302]
### 问题描述 Please describe your issue

![企业微信截图_16636649939563](https://user-images.githubusercontent.com/16071449/191220179-3b3c5831-7726-44dc-83c2-a81547cd7397.png)
![企业微信截图_16636650182294](https://user-images.githubusercontent.com/16071449/191220239-4f4f533a-eb31-4c44-b26c-64835e7fb9e5.png)

另外这个 label 参数是否多余? triplet loss 公式一般就不需要 label 表示： `loss = max(0, margin + (input - other))`

**Comment by yt605155624** [comment]
请问具体注释在哪，麻烦你发现代码链接行

**Comment by jiaqianjing** [comment]
> 请问具体注释在哪，麻烦你发现代码链接行
https://github.com/PaddlePaddle/Paddle/blob/3845229dc8d1b5279e3f7c28ed333a6a9ba157b5/python/paddle/nn/functional/loss.py#L1088
另外我关于这个提了一个 PR： https://github.com/PaddlePaddle/Paddle/pull/46306

**Comment by wawltor** [comment]
您好，非常感谢您给我们贡献代码，代码已合入！ https://github.com/PaddlePaddle/Paddle/pull/46306
